csgc-jekyll-src/resources.md

```diff
@@ -16,6 +16,7 @@ permalink: /resources/
 * [Center for Instructional Development and Educational Research](/cider/): These are some useful resources for students who expect to be teaching a class.
 * [Computer Science Compact for the Advisor relationship](https://gpc.cs.vt.edu/shaffer/CSCompact.pdf): A guide to help describe an aspirational relationship between a student and their Advisor.
 * [List of faculty seeking students](https://docs.google.com/spreadsheets/d/1BD1zxg6qMXW01GLofcS1ny034B7aQVT9zjuZF1CUgBQ): This spreadsheet contains the current status of faculty members in the department, showing whether or not they are currently open to advising more graduate stduents.
+* [CSRankings](http://csrankings.org): How Virginia Tech compares to other universities in various areas.
```

Comment:
2 thoughts:

csrankings.org is typically useful to _prospective_ students, who are trying to figure out where their research interests would be most at home (putting aside the fact that many research areas are totally ignored at this website...). Though I could see this being useful for students trying to figure out where they should try to get their work published. Which brings me to my next point.

> How Virginia Tech compares to other universities in various areas

I'm not sure that this is the best description of this website's utility.  Maybe change this to something similar to what I've described above.

Reply:
> I'm not sure that this is the best description of this website's utility. Maybe change this to something similar to what I've described above.

+1. Counting papers is useful only as a first approximation.